src/modules/GameConstants.ts

```diff
@@ -835,7 +835,7 @@ export const KalosDungeons = [
     'Glittering Cave',
     'Reflection Cave',
     // 'Tower of Mastery',
-    'Azure Bay', // Should really be a route
+    // 'Azure Bay', Should really be a route
```

Comment:
We can just completely remove this, don't think it will ever be needed to bring it back, as it is now a route.

Will also need to remove it from the `dungeonList`
```suggestion
```